### PR TITLE
Wikidata and other linked data in JSON api

### LINKED ISSUE #391: Expose Wikidata IDs in the API
Expose Wikidata IDs in the API to let reusers do smart things with it (access to Wikidata data, other db…)

### PR DESCRIPTION
**Pull request by hangy**
``` 
{
   "count":4,
   "tags":[
      {
         "id":"en:pizza-with-ham-and-cheese",
         "name":"Pizza with ham and cheese",
         "products":1,
         "url":"http://world.productopener.localhost/category/pizza-with-ham-and-cheese"
      },
      {
         "id":"en:pizzas",
         "name":"Pizzas",
         "linkeddata":{
            "wikidata:en":"Q177"
         },
         "products":1,
         "url":"http://world.productopener.localhost/category/pizzas"
      },
      {
         "id":"en:pizzas-pies-and-quiches",
         "name":"Pizzas
pies and quiches",
         "linkeddata":{
            "pnns_group_2:en":"Pizza pies and
quiche"
         },
         "products":1,
         "url":"http://world.productopener.localhost/category/pizzas-pies-and-quiches"
      },
      {
         "id":"en:meals",
         "name":"Meals",
         "linkeddata":{
            "pnns_group_2:en":"One-dish
meals"
         },
         "products":1,
         "url":"http://world.productopener.localhost/category/meals"
      }
   ]
}
```

Fixes #391

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -1345,6 +1346,10 @@ sub display_list_of_tags($$) {
 				url => format_subdomain($subdomain) . $product_link,
 				products => $products + 0, # + 0 to make the value numeric
 			};
+			
+			if (defined $linkeddata) {
+				$tagentry->{linkeddata} = $linkeddata;
```
Comment (hangy) [inline lib/ProductOpener/Display.pm]: I'm not sure if `linkeddata` is the best name for this property, though. The ProductOpener source calls the hash `$properties` , but a `properties` property could be too generic. Any ideas?

Comment (teolemon) [reply lib/ProductOpener/Display.pm]: same_as_wikidata_qid
same_as_url  ?
what do you think @stephanegigandet

**Comment by teolemon** [comment]
Having a way to write a Wikidata Qid somewhere would be swell. Is there a mediawiki API we could leverage to edit the Global Taxonomies page?

**Comment by hangy** [comment]
The current approach would include every additional property from the taxonomies.

File: lib/ProductOpener/Display.pm (hunk 1 above)
Comment (hangy) [reply lib/ProductOpener/Display.pm]: Maybe just `sameAs`, as it's used in [schema.org](http://schema.org/sameAs)?

**Comment by hangy** [comment]
> Having a way to write a Wikidata Qid somewhere would be swell. Is there a mediawiki API we could leverage to edit the Global Taxonomies page?

We could probably find an easy way to query Wikidata, but I parsing the taxonomy and adding the modified content to the OFF Wiki might be more complex. I think there was a related proposal in the GSoC list. Out of scope for this PR. 😄

**Pull request merged**